dart/lib/src/protocol/sentry_span.dart

```diff
@@ -44,14 +46,23 @@ class SentrySpan extends ISentrySpan {
     if (status != null) {
       _status = status;
     }
-    _timestamp = endTimestamp ?? getUtcDateTime();
+
+    if (endTimestamp?.isBefore(_startTimestamp) ?? false) {
```

Comment:
```suggestion
    if (endTimestamp?.toUtc()?.isBefore(_startTimestamp) ?? false) {
```

I guess this is needed?

Reply:
`_startTimestamp` is already UTC at this point, right?

Reply:
I wasn't sure about that and just checked it now, and date comparisons ([isAtSameMomentAs](https://api.dart.dev/stable/2.16.0/dart-core/DateTime/isAtSameMomentAs.html), [isBefore](https://api.dart.dev/stable/2.16.0/dart-core/DateTime/isBefore.html), [isAfter](https://api.dart.dev/stable/2.16.0/dart-core/DateTime/isAfter.html) etc.) look time zone independent.

For `isBefore`:
> Returns true if [this](https://api.dart.dev/stable/2.16.0/dart-core/DateTime-class.html) occurs before other.
> The comparison is independent of whether the time is in UTC or in the local time zone.

Reply:
> 

Yes, all inputs are converted to UTC before assigning. See [sentry_span_test.dart](https://github.com/fatihergin/sentry-dart/blob/feat/allow-to-set-sentryspan-starttimestamp-explicitly/dart/test/sentry_span_test.dart#L11).